netket/models/mlp.py

```diff
@@ -0,0 +1,94 @@
+from typing import Any, Tuple
+
+import numpy as np
+
+from flax import linen as nn
+from jax.nn.initializers import lecun_normal, zeros
+
+from netket.utils import deprecate_dtype
+from netket.utils.types import NNInitFunc
+from netket import nn as nknn
+
+default_kernel_init = lecun_normal()
+default_bias_init = zeros
+
+
+@deprecate_dtype
+class MLP(nn.Module):
```

Comment:
Can you add a class do string with some reference and/or more long form explanation of the various attributes/options?